src/audio/selector/selector_generic.c

```diff
@@ -18,6 +18,7 @@
 #include <stddef.h>
```

Comment:
Same question.

Reply:
ok, I split moving grouping function to separate commit